app/scripts/models/auth_brokers/fx-sync-channel.js

```diff
@@ -131,6 +134,30 @@ define(function (require, exports, module) {
         .then(() => proto.beforeSignUpConfirmationPoll.call(this, account));
     },
 
+    afterSignInConfirmationPoll (account) {
+      return proto.afterSignInConfirmationPoll.call(this, account)
+        .then((behavior) => {
+          if (this.hasCapability('sendAfterSignInConfirmationPollNotice')) {
+            const loginData = this._getLoginData(account);
+            return this.send(this.getCommand('EMAIL_VERIFIED'), loginData)
```

Comment:
Once upon a time we had plans to replace the current sign-in confirmation flow with other approaches, Google Authenticator etc. Would this command name seem weird if/when that happens?